README.md

```diff
@@ -72,6 +72,34 @@ If you're not using precompiled libs, build the projects in *__BUILD_BEFORE* fol
 `Build > Build Solution`
 
 
+## Building on Windows (MinGW):
+
+1) Install packages
+- `pacman -S base-devel mingw-w64-x86_64-toolchain mingw-w64-x86_64-cmake mingw-w64-x86_64-yasm mingw-w64-x86_64-python2 mingw-w64-x86_64-ntldd-git mingw-w64-x86_64-qt5 mingw-w64-x86_64-openal mingw-w64-x86_64-glew git`
+2) Clone repository
+- `git clone https://github.com/RPCS3/rpcs3.git`
+3) Update submodules
+- `cd rpcs3`
+- `git submodule update --init`
+- `cd ..`
+4) Configure and compile rpcs3
+- `mkdir rpcs3_build && cd rpcs3_build`
+- `cmake -G "MSYS Makefiles" -DCMAKE_MAKE_PROGRAM=mingw32-make -DCMAKE_BUILD_TYPE=Release ../rpcs3/`
```

Comment:
Just remembered that `DCMAKE_BUILD_TYPE` is set to `Release` by default, so you don't need to add this.